translator/translate/metrics/global_dimensions/globalDimensions_test.go

```diff
@@ -0,0 +1,81 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+package global_dimensions
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGlobalDimensions(t *testing.T) {
+	e := new(globalDimensions)
+	var input interface{}
+	err := json.Unmarshal([]byte(`{
+      "global_dimensions": {
+				"Environment": "test",
```

Comment:
Another key I would imagine to be invalid is `number` or at least out of bound string.

Reply:
Yes, this is true _but_ the schema should give us protection from this shouldn't it? I'm moderately happy to add checks for different JSON data types as this doesn't increase complexity very much.

By "out of bound string" do you mean too long? The other surrounding translators don't seem to have these kind of checks in place but if you really want them I could add them.

Reply:
I'm indifferent about adding this